library/src/com/google/maps/android/geojson/GeoJsonLayer.java

```diff
@@ -0,0 +1,251 @@
+package com.google.maps.android.geojson;
+
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.LatLngBounds;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import android.content.Context;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+
+/**
+ * A class that allows the developer import GeoJSON data, style it and interact with the imported
+ * data.
+ */
+public class GeoJsonLayer {
+
+    private final GeoJsonRenderer mRenderer;
+
+    private final GeoJsonParser mParser;
+
+    private GeoJsonPointStyle mDefaultPointStyle;
+
+    private GeoJsonLineStringStyle mDefaultLineStringStyle;
+
+    private GeoJsonPolygonStyle mDefaultPolygonStyle;
+
+    private LatLngBounds mBoundingBox;
+
+    /**
+     * Creates a new GeoJsonLayer object
+     *
+     * @param map         GoogleMap object
+     * @param geoJsonFile JSONObject to parse GeoJSON data from
+     */
+    public GeoJsonLayer(GoogleMap map, JSONObject geoJsonFile) {
+        mDefaultPointStyle = new GeoJsonPointStyle();
+        mDefaultLineStringStyle = new GeoJsonLineStringStyle();
+        mDefaultPolygonStyle = new GeoJsonPolygonStyle();
+        mBoundingBox = null;
+        mRenderer = new GeoJsonRenderer(map);
+        mParser = new GeoJsonParser(geoJsonFile);
+        // Assign GeoJSON bounding box for FeatureCollection
+        mBoundingBox = mParser.getBoundingBox();
+    }
+
+    /**
+     * Creates a new GeoJsonLayer object
+     *
+     * @param map        GoogleMap object
```

Comment:
"GoogleMap object" is implied by the method signature. Same for anywhere you've used "<Type> object" in the doc. For this one, you can say something like "The map where the layer is to be rendered."